ircd/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ configMapGenerator:
   - files/ircd.motd
   - files/ircd.yaml
 images:
-  - name: oragono/oragono:latest
```

Comment:
You don't change this line unless you also change the deployment.

Just the digest change is sufficient.